### PR TITLE
Adding support for multi orders placement.

### PR DESCRIPTION
**Pull request by PeGilo**
Added 2 methods to IMatchingEngineClient:
- PlaceMultiLimitOrder
- CancelMultiLimitOrder

### CODE REVIEW

File: src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderModel.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Lykke.MatchingEngine.Connector.Abstractions.Models
+{
+    public class MultiLimitOrderModel
+    {
+        public string Id { get; set; } 
+        public string ClientId { get; set; }
+        public string AssetId { get; set; }
+        public bool CancelPreviousOrders { get; set; }
+        public MultiOrderItemModel[] Orders { get; set; }
```
Comment (MikhailGerasimov) [inline src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderModel.cs]: Can be IReadOnlyList

File: src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderResponseModel.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{
+    public class MultiLimitOrderResponse
+    {
+        public string Id { get; set; }
+        public MeStatusCodes Status { get; set; }
+        public string StatusReason { get; set; }
+        public string Message { get; set; }
+        public string TransactionId { get; set; }
+        public string AssetPairId { get; set; }
+        public LimitOrderStatusModel[] Statuses { get; set; }
```
Comment (MikhailGerasimov) [inline src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderResponseModel.cs]: Can be IReadOnliList

File: src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderModel.cs (hunk 1 above)
Comment (PeGilo) [reply src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderModel.cs]: Fixed.

File: src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderResponseModel.cs (hunk 2 above)
Comment (PeGilo) [reply src/Lykke.MatchingEngine.Connector.Abstractions/Models/MultiLimitOrderResponseModel.cs]: Fixed.

**Review by tarurar — APPROVED** [review]

**Review by NovichikhinAlexey — APPROVED** [review]

**Pull request merged**